test/functional/wallet_basic.py

```diff
@@ -267,6 +267,20 @@ def run_test(self):
         assert_equal(self.nodes[2].getbalance(), node_2_bal)
         node_0_bal = self.check_fee_amount(self.nodes[0].getbalance(), node_0_bal + Decimal('10'), fee_per_byte, self.get_vsize(self.nodes[2].gettransaction(txid)['hex']))
 
+        # Sendmany 5 BTC to two address with subtracting fee from both
+        a0 = self.nodes[0].getnewaddress()
+        a1 = self.nodes[0].getnewaddress()
+        txid = self.nodes[2].sendmany('', {a0: 5, a1: 5}, 0, "", [a0, a1])
```

Comment:
Feel free to ignore, just a simple suggestion to make it friendly to read:
```suggestion
        txid = self.nodes[2].sendmany(dummy='', amounts={a0: 5, a1: 5}, minconf=0, comment="", subtractfeefrom=[a0, a1])
```

Reply:
@brunoerg I think that's better, thanks for the suggestion.